cl/recap/tasks.py

```diff
@@ -373,7 +373,7 @@ def process_recap_zip(self, pk):
     mark_pq_status(pq, "", PROCESSING_STATUS.IN_PROGRESS)
 
     logger.info("Processing RECAP zip (debug is: %s): %s", pq.debug, pq)
-    with ZipFile(pq.filepath_local.path, "r") as archive:
+    with ZipFile(pq.filepath_local.name, "r") as archive:
```

Comment:
This surprises me. Can you share the background of this change? It looks like it'd break things, but an old memory tells me that it's needed for AWS for some reason...or something?

Reply:
hmm... I was worried about that... let me throw a testing ... solution at the wall

Reply:
I added something for you to review.